### ISSUE TITLE
Add emoji selector on profile custom fields

### ISSUE THREAD

**Issue by roipoussiere** [tootsuite/mastodon#7793]
Since it's possible to insert emoji in profile custom field values, an emoji picker could be added to simplify the process, like when creating a toot.

Here is a suggestion:

![emoji-selector](https://user-images.githubusercontent.com/1665542/41303426-8b4c391a-6e6d-11e8-9ac1-e55018942d8b.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by rugk** [comment]
I agree. If you are in desperate need for an emoji picker [I also have one](https://addons.mozilla.org/firefox/addon/mastodon-simplified-federation/?src=external-github-issue) though… :wink: (for your whole browser then, but it's based on the same emoji-picker-lib as Mastodon uses, actually.)

**Comment by rugk** [comment]
still a neat idea IMHO

**Comment by ChaosExAnima** [comment]
should be resolved with the new profile editor in 4.6, specifically #38066. all text fields (name, bio, field label + value) have an emoji picker button.
